src/libs/deployless/deployless.ts

```diff
@@ -19,6 +19,8 @@ const deployErrorSig = '0xb4f54111'
 const errorSig = '0x08c379a0'
 // Signature of Panic(uint256)
 const panicSig = '0x4e487b71'
+// uniswap swap expired
+const expiredSig = '0x5bf6f916'
```

Comment:
this error should NOT be parsed here, it's way too specific

Or we should make another library for error parsing that deployless uses

Reply:
I haven't forgotten about this, will fix